README.md

```diff
@@ -1,2 +1,34 @@
-TODO
-### Che server
\ No newline at end of file
+[![Next build Status](https://github.com/eclipse-che/che-theia/actions/workflows/next-build.yml/badge.svg)]
+[![Release build Status](https://github.com/eclipse-che/che-theia/actions/workflows/release.yml/badge.svg)]
+
+# What is Che server
+Che Server is a core component of the Eclipse Che. The component is responsibe for creation and managing of Che workspaces.
+
+# Project structure
+Che Server is mostly a Java web application deployed on a Apache Tomcat server in a container. The primary root module ['pom.xml'](https://github.com/eclipse-che/che-server/tree/main/pom.xml) lists all the java submodules. In the ['dockerfiles'](https://github.com/eclipse-che/che-server/tree/main/dockerfiles) directory contains container image for Che Server, as well as additional images, such as container for PostgreSQL DB. Additionally, there is a separate ['typescript-dto'](https://github.com/eclipse-che/che-server/tree/main/typescript-dto) module, that provides DTO objects for typescript projects that may depend on Che Server, such as Che Theia.
```

Comment:
Instead of one long paragraph, I would maybe write it as bullet list like CI part

Reply:
especially sentence about dockerfiles feels clumsy. the directory definitely does not containe conainer images. It contains only dockerfiles to build these images.

Reply:
reworked